solidity/contracts/utility/Owned.sol

```diff
@@ -37,15 +42,15 @@ contract Owned is IOwned {
       * @param _newOwner    new contract owner
     */
     function transferOwnership(address _newOwner) public ownerOnly {
-        require(_newOwner != owner);
+        require(_newOwner != owner, "ERR_SAME_OWNER");
         newOwner = _newOwner;
     }
 
     /**
       * @dev used by a new owner to accept an ownership transfer
     */
     function acceptOwnership() public {
-        require(msg.sender == newOwner);
+        require(msg.sender == newOwner, "ERR_ACCESS_DENIED");

```

Comment:
@yudilevi : We have two different types of `ERR_ACCESS_DENIED` in this contract (not critical, because they are essentially the same, right?).

Reply:
what do you mean by 2 types?

Reply:
@yudilevi: I mean that the same error-message is used in the `onlyOwner` modifier (which is used in a bunch of different places).

Reply:
ah yeah that's fine